01_Data/src/layers/sequelize/model/Authorization/LocalListVersion.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2023 S44, LLC
+// Copyright Contributors to the CitrineOS Project
+//
+// SPDX-License-Identifier: Apache 2.0
+
+import { Namespace, } from "@citrineos/base";
+import { CustomDataType } from "@citrineos/base/src/ocpp/model/types/SendLocalListRequest";
+import { BelongsToMany, Column, DataType, Model, Table } from 'sequelize-typescript';
+import { LocalListAuthorization } from ".";
+import { LocalListVersionAuthorization } from "./LocalListVersionAuthorization";
+
+@Table
+export class LocalListVersion extends Model {
+    static readonly MODEL_NAME: string = Namespace.LocalListVersion;
+
+    @Column({
+      unique: true,
+    })
+    declare stationId: string;
+    
+    @Column(DataType.DECIMAL)
```

Comment:
Should `versionNumber` be integer based on the protocol in part2, 1.23.2? Same question for the versionNumber in SendLocalList table.

Reply:
Good point, fixed!